### ISSUE TITLE
Request: Lime

### ISSUE THREAD

**Issue by DerBraky** [simple-icons/simple-icons#11983]
### Brand Name

Lime

### Website

https://www.li.me

### Popularity Metric

- Similarweb Global Rank: [86,188](https://www.similarweb.com/website/li.me) 

### Official Resources for Icon and Color

- SVG: https://drive.google.com/file/d/1trEI0INLMCBYnemoc4O-zDTLm5H8s9cc/view (Press Kit)
- Color: `#00dd00`, from logo

### Additional Comments

Press Kit: https://drive.google.com/drive/folders/1nXSF68Y7_MXHzEpACILve_CHJ7jYGyvC (via https://www.li.me/about/press)

**Comment by uncenter** [comment]
For the source we should link to https://www.li.me/about/press as the link to the Drive press kit may change.

**Comment by Anonav0** [comment]
Could you assign me this? @uncenter

**Comment by uncenter** [comment]
We don't assign issues, feel free to raise a PR though. See our guidelines at https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#adding-or-updating-an-icon.

**Comment by uncenter** [comment]
Nevermind actually, someone else already opened a PR.

**Comment by Anonav0** [comment]
Yes I checked that PR request earlier. I saw a lot of problem related to that. I have forked a version the repo and was trying to add the logo myself reading the docs, but I'm facing problem while testing. I was using inkscape for the extraction of the logo and SVGO to optimize it. But it seems like due to the strict 'monochromatic' check. My version of the logo has fill. 

Maybe I'm doing something wrong or should I not use inkscape all together to do this. But it seems like most of the hard hitting job is done already only the monochromatic logo task is the problem.

**Comment by uncenter** [comment]
> Maybe I'm doing something wrong or should I not use inkscape all together to do this. But it seems like most of the hard hitting job is done already only the monochromatic logo task is the problem.

You can export the icon from Inkscape and then go to the raw SVG text and remove the fill/style attribute.

**Comment by EvanPrograms** [comment]
Here is my pull request: https://github.com/simple-icons/simple-icons/pull/12490

**Comment by Garv978** [comment]
Hi! I'm looking to make my first open-source contribution and would love to tackle this bug is this done ?
